packages/@lwc/module-resolver/src/__tests__/module-errors.spec.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import path from 'path';
+import { resolveModule } from '../index';
+
+describe('resolve individual module', () => {
+    test('throw when no modules are empty', () => {
+        function run() {
+            const customImporter = path.join(__dirname, 'fixtures/errors/empty/empty.js');
+            const expectedImportee = 'empty';
+
+            resolveModule(expectedImportee, customImporter);
+        }
+
+        expect(run).toThrow(
+            new Error('Unable to resolve empty: No ModuleRecords have been defined')
+        );
+    });
+
+    test('throw when alias has no path', () => {
+        function run() {
+            const customImporter = path.join(
+                __dirname,
+                'fixtures/errors/invalid-alias/invalid-alias.js'
+            );
+            const expectedImportee = 'invalid-alias';
+
+            resolveModule(expectedImportee, customImporter);
+        }
+
+        expect(run).toThrow();
+
+        try {
+            run();
+        } catch (e) {
+            expect(e.message).toContain('Unable to resolve invalid-alias from');
```

Comment:
The message assertion should be done in the `toThrow` like in the first test case. 
This test should also assert the entire message to check the path used in the error message. You can use a regular expression to match against the path without hardcoding your own root directory.